cmd/ketch/builder_list.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+)
+
+const builderListHelp = `
+List CNCF registered builders
+`
+
+type BuilderListEntry struct {
+	Vendor      string
+	Image       string
+	Description string
+}
+
+var builderList = []BuilderListEntry{
```

Comment:
I would prefer to go with that approach, but since pack itself doesn't publicly expose any of the methods to get the recommended builders I don't think it is an option. Any of the viable options won't allow us to change how the command is written and requires some more overhead to provide additional functionality that I feel is out of the scope of this issue.